app/models/detector/citation.rb

```diff
@@ -69,6 +72,12 @@ def initialize(phrase)
       @score = calculate_score
     end
 
+    def detections
```

Comment:
Should this method have tests in the rest of the model test file?

Reply:
Additional commit addresses this by adding explicit tests for this new method.